.circleci/config.yml

```diff
@@ -55,10 +84,44 @@ jobs:
           PROJECT="circleci/aws-code-deploy-orb-test/build"
           curl -X POST $BASE/$PROJECT?circle-token=$CIRCLECI_API_TOKEN
 
-  release:
+  slack-notify-hold:
+    machine: true
```

Comment:
Feels heavy weight to provisioning a VM for a slack message.

Any reason it cant be the final step on the previous job?